src/test/java/com/logigear/BookTicketTest.java

```diff
@@ -43,4 +45,32 @@ public void TC14() {
         Assert.assertEquals(actualErrorMsg, expectedErrorMsg, "Error message fails to display! ");
         Assert.assertEquals(actualTicketAmountErrorMsg, expectedTicketAmountErrorMsg, "Ticket amount error fails to display!");
     }
+
+    @Test(description = "TC15 - User can open Book ticket page by click on Book ticket link in Ticket price")
+    public void TC15() {
+        LoggerHelper.startTestCase("TC15 - User can open Book ticket page by click on Book ticket link in Ticket price");
+
+        String departFrom = "Sài Gòn";
+        String arriveAt = "Nha Trang";
+        String seatType = "Hard seat";
+        Account account = new Account(DataHelper.getRandomValidEmail(), DataHelper.getRandomValidPassword(), DataHelper.getRandomValidPid());
+
+        homePage.goToRegisterPage();
+        registerPage.register(account);
+        homePage.goToLoginPage();
+        loginPage.login(account);
+        homePage.goToTicketPricePage();
+        ticketPricePage.clickOnCheckPrice(departFrom, arriveAt);
+        ticketPricePage.clickOnBookTicket(seatType);
+
+        BrowserHelper.scrollPage();
```

Comment:
why do we need scrollPage?

Reply:
in my TC, the book ticket page not auto-scroll so I need a scrolling page to check the information book ticket. and assert actual with expected result